debian/changelog

```diff
@@ -1,14 +1,13 @@
-ubuntu-image (3.10.4) UNRELEASED; urgency=medium
-
-  * i/statemachine: add riscv64-efi GRUB target (LP: #2134449)
-
- -- Heinrich Schuchardt <heinrich.schuchardt@canonical.com>  Tue, 09 Dec 2025 19:45:26 +0100
-
 ubuntu-image (3.10.3) UNRELEASED; urgency=medium
 
+  [Maciej Borzecki]
   * Pass snapdusergo build tag.
 
- -- Maciej Borzecki <maciej.borzecki@canonical.com>  Thu, 27 Nov 2025 12:48:12 +0000
+  [Alexis Cellier]
+  * Use divertion instead of backup/restore to prepare chroot, avoiding
+    restoring old file version.
+
```

Comment:
Another PR was merged in the meantime, so you might want to rebase (or merge existing changes from main) and move this entry under the other release.

Reply:
Fixed